tests/tck/features/aggregate/Agg.feature

```diff
@@ -691,12 +691,12 @@ Feature: Basic Aggregate and GroupBy
       """
       GO FROM "Tim Duncan" OVER like YIELD count(*)
       """
-    Then a SemanticError should be raised at runtime: `count(*)' is not support in go sentence.
+    Then a SyntaxError should be raised at runtime: Invalid use of aggregating function in yield clause. near `count(*)'
```

Comment:
Why disable this usage?

Reply:
This  was not allowed before, and using  agg  funciton in ngql 's  yield clause  is meaningless